Paymetheus.Rpc/WalletClient.cs

```diff
@@ -314,6 +332,77 @@ public void CancelRequests()
             return Tuple.Create(signedTransaction, complete);
         }
 
+        public async Task<List<Blake256Hash>> PurchaseTicketsAsync(Account account, Amount spendLimit, 
+            int reqConfs, Address ticketAddress, uint number, Address poolAddress, double poolFees, 
+            uint expiry, Amount txFee, Amount ticketFee, string passphrase)
```

Comment:
I agree that it is gross. However, it is currently done this way to comply with the structure of the gRPC call, and this itself was based on the JSON call purchaseticket which is a mess. In the future I think logic should rather be added to create tickets directly in the wallet using UTXOs, as is currently done with transaction generation. The purpose of the current PR is to add a simple version of the functionality so that users have access to ticket purchasing. In the future, we can modify the backend to make it clearer and more robust, and no longer call the gRPC function since it has so many issues.